### PR TITLE
fix: [PLATO-350] Netlify deploy button

### PR DESCRIPTION
**Pull request by mattvanvoorst-contentful**
## Purpose of PR

- Added a button to deploy the repository from Netlify, adjusted the readme accordingly

https://contentful.atlassian.net/browse/PLATO-350

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,34 @@
 # Contentful Marketing Starter Template
 
-A Marketing Starter Template powered by Next.js & Contentful, pre-designed with optimized & adjustable pages, components, and data management.
+A Marketing Starter Template powered by Next.js & Contentful, pre-designed with optimized & adjustable pages,
+components, and data management.
```
Comment (AdrianLThomas) [inline README.md:4]: did you mean to add a new line for L4?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
 
 ![The homepage of the Marketing Starter Template](marketing-starter-template.jpg "The homepage of the Marketing Starter Template")
 
 $~$
 
 ## What is Contentful?
 
-[Contentful](https://www.contentful.com/) provides content infrastructure for digital teams to power websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management, and delivery APIs, and a customizable web app that enables developers and content creators to ship their products faster.
+[Contentful](https://www.contentful.com/) provides content infrastructure for digital teams to power websites, apps, and
+devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for
```
Comment (AdrianLThomas) [inline README.md:13]: same here?

File: README.md (hunk 1 above)
Comment (mattvanvoorst-contentful) [reply README.md:4]: That's just a soft line break

File: README.md (hunk 2 above)
Comment (mattvanvoorst-contentful) [reply README.md:13]: That's just a soft line break

**Review by AdrianLThomas — APPROVED** [review]

**Review by missating — APPROVED** [review]

**Pull request merged**